### PR TITLE
Update pagination setting

### PR DESCRIPTION
**Pull request by Monalisa98927**
* remove the former setting button and modal
* create a new pagination setting similar to Wikidata; three options - view 20/50/100 per page
* add "Go to xxx Page"

### CODE REVIEW

**Comment by Monalisa98927** [comment]
I just updated the pagination setting. Maybe the review can be done after the project meeting so it's not rushed.

**Comment by dchiller** [comment]
This looks great! I have only two comments:

image

I think it would be better to have "Go to Page XX" rather than "Go To XX Page". It's only European languages, so I'd be curious what other's find, but it looks better to me that way in English and when translated to Spanish and French than the way it currently is.

image

I wonder if the numbers here could either have more contrast or have some kind of divider ... they run together a bit for me here.

**Comment by Monalisa98927** [comment]
Thank you for the advice. I'm going to use 'Go to Page XXX'. Maybe I can add vertical lines between page numbers like [Wikidata](https://www.wikidata.org/w/index.php?search=musical+instrument&title=Special%3ASearch&profile=advanced&fulltext=1&ns0=1&ns120=1) does. 
image

**Comment by Monalisa98927** [comment]
Now the paginator looks like this
image

**Comment by Monalisa98927** [comment]
Have updated the pagination setting. Now it looks like this
image

**Comment by dchiller** [comment]
Looks great!

File: web-app/django/VIM/apps/instruments/templates/instruments/index.html [hunk 1]
```diff
@@ -93,10 +94,9 @@ <h4 class="ms-3 me-2 my-auto"><small>INSTRUMENT LIST</small></h4>
       </div>
 
       <div class="d-flex">
-        <button type="button" class="btn me-1 px-1 py-0 justify-content-center display-btn" id="page-setting-btn" data-bs-toggle="modal" data-bs-target="#paginationModal">
+        {% comment %} <button type="button" class="btn me-1 px-1 py-0 justify-content-center display-btn" id="page-setting-btn" data-bs-toggle="modal" data-bs-target="#paginationModal">
```
Comment (dchiller) [inline web-app/django/VIM/apps/instruments/templates/instruments/index.html:97]: Do you want to keep this comment in?

**Review by dchiller — APPROVED** [review]

File: web-app/django/VIM/apps/instruments/templates/instruments/index.html (hunk 1 above)
Comment (Monalisa98927) [reply web-app/django/VIM/apps/instruments/templates/instruments/index.html:97]: It seems that we don't need the setting button at present, I'll delete it.

**Pull request merged**